src/server/api/admin_api.py

```diff
@@ -99,33 +108,35 @@ def checkStatus():
         for row in query_result:
             results = dict(row)
         return jsonify(results)
-'''
+"""
 
 
-@admin_api.route('/api/statistics', methods=['GET'])
+@admin_api.route("/api/statistics", methods=["GET"])
 def listStatistics():
     try:
-        last_execution_file = open(LOGS_PATH + 'last_execution.json', 'r')
+        last_execution_file = open(LOGS_PATH + "last_execution.json", "r")
         last_execution_details = json.loads(last_execution_file.read())
         last_execution_file.close()
 
     except Exception as e:
-        return abort(404)
+        return jsonify(None)
```

Comment:
What if we have a real exception?  Can't we do a simple file test rather than catch and ignore the exception?

Reply:
I have no problem with that idea but I'm not sure how we'd handle anything else. If the JSON was bad and we got a JSONDecodeError we'd probably want to log it but we couldn't pass it on.

Reply:
Correct, that should be logged and a 500 error returned so you know there is a server error.  I'm not sure how flask handles exceptions bubbling up, but worth a look

Reply:
I'm now logging and returning a 500 if the file is missing or not valid JSON. I've tested each case.
I'm also creating a minimal valid (null) last_execution.json file if there's not one already, so if we get an error here it's a real problem. 
I'll create an issue to notify the client folks about the possibility of a 500.